src/service-worker.js

```diff
@@ -1,4 +1,3 @@
 self.addEventListener('activate', function () {
   // Do nothing!
```

Comment:
See, I though this had to be there for the Android app-install prompt (see https://github.com/umts/pvta-multiplatform/issues/283#issuecomment-289164011).

But now, I wonder if we misread the documentation?  From "[Native App Install Banners](https://developers.google.com/web/fundamentals/engage-and-retain/app-install-banners/#native_app_install_banners)":

>  The criteria is similar to the Web App install banner except for the need of a service worker...

@akaplo, is there any other reason for this?